docs/source/gcp/gettingstarted.rst

```diff
@@ -119,8 +118,9 @@ Next, run the following command to execute the policy with Custodian:
 
 .. code-block:: bash
 
-    custodian run --output-dir=. custodian.yml
+    GOOGLE_CLOUD_PROJECT="project-id" custodian run --output-dir=. custodian.yml
```

Comment:
a random aside, we generally want try to use the env vars that the cli chooses, but google is internally schizophrenic and uses different env vars by different tools. The cli name squats on CLOUDSDK_CORE_PROJECT and the sdk uses several names though GOOGLE_CLOUD_PROJECT appears to be the latest naming flavor of the month.